File: iniciante/python/1045.py
# -*- coding: utf-8 -*-

A, B, C = input().split(" ")
A = float(A)
B = float(B)
C = float(C)

if B > A:
    A, B = B, A
    
if C > A:
    A, C = C, A

if A >= B + C:
    print("NAO FORMA TRIANGULO")

if A**2 == B**2 + C**2:
    print("TRIANGULO RETANGULO")
    
if (A**2 > B**2 + C**2) and (A < B+C):
    print("TRIANGULO OBTUSANGULO")

if  A**2 < B**2 + C**2:
    print("TRIANGULO ACUTANGULO")

if A == B and A == C:
    print("TRIANGULO EQUILATERO")

if (A == B and A != C) or (A == C and A != B) or (B == C and A != B) and(A < B+C):
    print("TRIANGULO ISOSCELES")

File: iniciante/c/1045.c
#include <stdio.h>
 
int main() {

	double A, B, C, aux;
	int i;
	
	scanf("%lf", &A);
	scanf("%lf", &B);
	scanf("%lf", &C);

	if (A < B){
		aux = A;
		A = B;
		B = aux;
	}
	if (A < C){
		aux = A;
		A = C;
		C = aux;
	}
	if (A >= B+C){
		printf("NAO FORMA TRIANGULO\n");
	}else{
		if ((A*A) == (B*B)+ (C*C)){
			printf("TRIANGULO RETANGULO\n");
		}
		if ((A*A) > (B*B)+ (C*C)){
			printf("TRIANGULO OBTUSANGULO\n");
		}
		if ((A*A)< (B*B)+ (C*C)){
			printf("TRIANGULO ACUTANGULO\n");
		}
		if (A == B && A==C){
			printf("TRIANGULO EQUILATERO\n");
		}
		if ((A == B && A!= C) ||(A == C && A!=B) || (B==C && B != A)){
			printf("TRIANGULO ISOSCELES\n");
		}
	}
 
    return 0;
}